android/src/main/java/com/rudderstack/android/storage/AndroidStorageImpl.kt

```diff
@@ -382,6 +386,14 @@ class AndroidStorageImpl(
         preferenceManager?.resetV1ExternalIds()
     }
 
+    override fun setBuild(build: Int) {
```

Comment:
same in preference. Change to VersionCode

Reply:
We have decided to keep `build` instead of `versionCode`.